src/Features/Core/Completion/ICompletionService.cs

```diff
@@ -12,6 +12,11 @@ namespace Microsoft.CodeAnalysis.Completion
     internal interface ICompletionService : ILanguageService
     {
         /// <summary>
+        /// The language that this <see cref="ICompletionService"/> targets.
+        /// </summary>
+        string LanguageName { get; }
```

Comment:
Why is this necessary in the interface?  Does anyone externally (i.e. not the impl of ICompletionService) use this?